v1alpha/schema/schema.proto

```diff
@@ -24,4 +24,5 @@ message Depends {
     string language = 1;
     string version_constraint = 2;
     repeated string scopes = 3;
+    string source_url = 4;
```

Comment:
Can we call this `ref` and then also add a `ref` field to the source object?

The thought is to allow this would allow the field to be a human readable value. When indexing repositories, it can be source url with branch name. When indexing an artifact repository, we could set it as a human readable version string.

Reply:
Have made the changes, please check if I've got them right. Are artifact repo names not human readable? If possible, can you share a sample response of these artifact repo discovery api's, just so I understand more of what we're doing there. But, also happy to wait for your changes on indexing them, to understand them myself.

Reply:
Unfortunately not in all cases. A very simple example is Nexus. They use what appear to be sha id's: https://help.sonatype.com/repomanager3/rest-and-integration-api/components-api#ComponentsAPI-GetComponent

Reply:
Okay, got it. Thank you.